src/App.js

```diff
@@ -1,13 +1,20 @@
 import Map from "./components/Map";
 
-import './App.css';
+import "./App.css";
 
 function App() {
```

Comment:
Javascript functions have a new way of being instanciated via arrow functions.
there will be less code to maintain if you use it for this example https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions